packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -390,10 +390,10 @@ export const useProductFullDetail = props => {
 
     // Normalization object for product details we need for rendering.
     const productDetails = {
-        description: product.description,
-        name: product.name,
+        ...product,
+        quantity: 1,
         price: productPrice,
-        sku: product.sku
+        selected_options: selectedOptionsArray
```

Comment:
Yes, all that is gonna change. This will be a helper hook and the consumer of this hook is required to send the data it needs to work in this case `wishlistItemOptions`.